src/depot/outdated/resolve_virtual.clj

```diff
@@ -45,6 +45,3 @@
           (spit file new-deps)
           (catch java.io.FileNotFoundException e
             (println "  [ERROR] Permission denied: " file)))))))
-
-#_
-(resolve-version 'cider/piggieback {:mvn/version "0.4.1-SNAPSHOT"} maven/standard-repos)
```

Comment:
Haha I meant to comment on this one in the first place, I really wasn't paying attention :upside_down_face: